src/main/java/org/apache/sling/feature/cpconverter/handlers/slinginitialcontent/BundleSlingInitialContentExtractor.java

```diff
@@ -103,11 +111,7 @@ static Version getModifiedOsgiVersion(@NotNull Version originalVersion) {
     }
 
     public void reset() {
-        parentFolderRepoInitHandler.reset();
-    }
-
-    public void addRepoInitExtension(@NotNull List<VaultPackageAssembler> assemblers, @NotNull FeaturesManager featureManager) throws IOException, ConverterException {
-        parentFolderRepoInitHandler.addRepoinitExtension(assemblers, featureManager);
+       
```

Comment:
this method is empty now.... and it is not inherited from any interface or superclass.... 

IMHO we can get rid of this altogether, right?
which also means (i looked for usages) that calling this `reset()` in the `ContentPackage2FeatureModelConverter` is no longer needed, which circles back to my comment about the converter class.... if i am not mistaken you could actually get rid of having the extractor passed to the converter altogether, no?